lab_7_llm/README.rst

```diff
@@ -1,729 +0,0 @@
-.. _lab_7_llm/README:
```

Comment:
Too radical an approach to PR cleanup. The lab 7 files must be rolled back to the current main branch

Reply:
Well, if _remove all files_ doesn't mean to remove all files...

Reply:
I agree, need to be more careful in the motives for action